### ISSUE TITLE
Office Management - PM-check-Toast jumps back into the toaster ;)

### ISSUE THREAD

**Issue by ArminGepardec** [Gepardec/mega#363]
Siehe #626 

Während man mit der Maus über dem PM-Check Icon hovert, wird der Toast mit dem Status aller PMs eingeblendet. Schiebt sich dieser unter die Maus, dann springt er immer Wieder zurück in den Toaster. 

TL;DR;
see video
https://user-images.githubusercontent.com/24893387/121491667-74852800-c9d6-11eb-90c5-12d806d02cc3.mov

**Comment by LeitnerMi** [comment]
@ArminGepardec @WernerBruckmueller Ist jetzt mit Click-Feature. Ist das schlimm, bzw. nicht gewollt?

https://user-images.githubusercontent.com/87299303/127836411-feb69f19-dfbd-4c36-b0e1-6a14f5e34165.mp4

**Comment by WernerBruckmueller** [comment]
Es ist nicht selbsterklärend, dass man hier klicken kann/soll.
Dieses Issue hat für mich eine sehr geringe Priorität, falls es hier keine rasche Lösung gibt.

@ArminGepardec @LeitnerMi videos dann bitte wieder entfernen.

**Comment by LeitnerMi** [comment]
Stimmt, aber war es ersichtlich, dass wenn man drüber hovered, ein BottomSheet kommt? Ich kanns so machen, dass wenn man drüber fährt, es sich öffnet, und wenn man rausklickt, es sich schließt.

**Comment by WernerBruckmueller** [comment]
Nicht ersichtlich, aber es hat sich wie ein ToolTip verhalten. und die User sind das Verhalten so gewohnt.

Möchte alle unnötigen klicks vermeiden.

**Comment by LeitnerMi** [comment]
Ist halt dann schwierig zu lösen, da: Beim drüber fahren poppt das Fenster auf. Beim Runtergehen wird's wieder klein. Irgendwann muss das Fenster dableiben, um das Problem mit dem Überschneiden zu lösen. Man könnte das vielleicht irgendwie Zeit gesteuert machen, aber da lass ich lieber einen vollwertigen Frontend-Developer ran ;)

**Comment by ArminGepardec** [comment]
Besteht die Möglichkeit einfach auf ein Tooltipartiges Verhalten umzustellen?
wenn man eine gewisse Verzögerung beim auslösen des Toasts hat, finde ich das hinausklicken aber auch in Ordnung. @WernerBruckmueller

**Comment by LeitnerMi** [comment]
Ja, das wäre glaube ich möglich, doch auch aufwendig(er). [Aber keine Ahnung ob Angular solche Sachen vielleicht sogar schon unterstützt]
Sprich: Falls du 1 Sekunde auf dem Icon hoverst, aktiviert sich das BottomSheet. Danach muss man halt rausklicken.

**Comment by WernerBruckmueller** [comment]
Nach Absprache mit @ArminGepardec soll aktuell keine Zeit investiert werden. Unser Wunsch ist, dass später ein eigenes UXD Thema gestartet wird, wo dieses Problem auch betrachtet werden soll.
